src/components/configuration-tree/configuration-tree.component.jsx

```diff
@@ -1,11 +1,5 @@
-/*
- * Copyright: Copyright 2023 SAP SE or an SAP affiliate company and cdc-tools-chrome-extension contributors
- * License: Apache-2.0
- */
-
-
```

Comment:
Add these comments 
/*
 * Copyright: Copyright 2023 SAP SE or an SAP affiliate company and cdc-tools-chrome-extension contributors
 * License: Apache-2.0
 */ 
 back on the file